about.html

```diff
@@ -0,0 +1,120 @@
+                  <ul class="mobnav">
+                      <li class="mobnav1" onclick="showhide()"><a class="moblink" href="./index.html">home</a></li><hr>
+                      <li class="mobnav1" onclick="showhide()"><a class="moblink" href="#">program</a></li><hr>
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">Join</a></li><hr>
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">Sponser</a></li><hr>
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">News</a></li><hr>
+                    </ul>
+              </div>
+                <div class="desk-links">
+                    <ul class="deskul">
+                        <li class="desklist"><a href="#" class="desklink"><i class="fa-brands fa-facebook-f"></i></a></li>
+                        <li class="desklist"><a href="#" class="desklink"><i class="fa-brands fa-twitter"></i></a></li>
+                        <li class="desklist"><a href="#" class="desklink">English</a></li>
+                        <li class="desklist"><a href="#" class="desklink">My page</a></li>
+                        <li class="desklist"><a href="#" class="desklink">Logout</a></li>
+                    </ul>
+                </div>
+                <div class="desk-nav">
+                    <div class="logolink">
+                      <a class="logolink1" href="#"><span class="desklogo">VR Odyssey</span></a>
+                    </div>
+                    <ul class="deskul2">
+                      <li class="desklist2"><a href="#" class="desklink1">About</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">program</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">Join</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">Sponser</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">News</a></li>
+                    </ul>
+                    <div class=desbutton><button class="deskbutton">VR Campaign</button></div>
+                </div>
+                <div class="home">
+                    <ul class="homeul">
+                      <li class="homeli"><a href="./index.html" class="desklink2"><i class="fa-solid fa-house homeicon"></a></i></li>
+                      <li class="homeli"><a href="#" class="desklink2"><i class="fa-solid fa-angle-right"></a></i></li>
+                      <li class="homeli"><a href="./about.html" class="desklink2">about</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <section class="page">
+            <p class="para">"Welcome to VR!"</p>
+            <div class="clip">
+                <h1 class="heading1">VR EVOLUTION:<br>THE NEXT CHAPTER 2023</h1>
+            </div>
```

Comment:
- [ ] While you do a great effort 😃 I see that you forgot to add the hr line below the heading here, so please try to add the hr line to be like the Behance Design 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225890865-d2461b27-a3b5-443d-94fc-690aa2c27f10.png)   |    ![image](https://user-images.githubusercontent.com/83360636/225890791-a06d89bc-db3f-4952-ae0f-716486fffdea.png)  |